src/PublicApiGeneratorTests/Field_types.cs

```diff
@@ -104,6 +121,12 @@ public class FieldWithMultipleGenericTypeParameters
         {
             public GenericTypeExtra<int, string, ComplexType> Field;
         }
+
+        public class FieldWithFunc
+        {
+            public readonly Func<string, string, string, IEnumerable<string>, string> FuncField = (a, b, c, d) => null;
+            public readonly Func<string, string, string, string> FuncField2;
```

Comment:
Note - this form of Func (without IEnumerable) works fine without changes in this PR **even with readonly**!